semantics/llvm-syntax.k

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) 2013-2014 K Team. All Rights Reserved.
+/*@
+
+This is the syntax of LLVM.
+
+This syntax requires a input LLVM program without any comments on it.
```

Comment:
Liyi, from now on I'd like to see no English mistakes in these public files.  If you are not sure about your English then have somebody review it all the time you commit something.